3.6.0a2 build fails on OS X 10.11.5 using python-build 20160130

```
$ pyenv install 3.6.0a2
Downloading Python-3.6.0a2.tgz...
-> https://www.python.org/ftp/python/3.6.0/Python-3.6.0a2.tgz
/usr/local/bin/python-build: line 230: pushd: Python-3.6.0a2: No such file or directory

BUILD FAILED (OS X 10.11.5 using python-build 20160130)

Inspect or clean up the working tree at /var/folders/mx/gn154d8j3254p213qkhd6n3mlnym4m/T/python-build.20160628093022.79796
Results logged to /var/folders/mx/gn154d8j3254p213qkhd6n3mlnym4m/T/python-build.20160628093022.79796.log

Last 10 log lines:
../Python-3.6.0a2/Doc/c-api/structures.rst: Path contains '..'
../Python-3.6.0a2/Doc/c-api/sys.rst: Path contains '..'
../Python-3.6.0a2/Doc/c-api/tuple.rst: Path contains '..'
../Python-3.6.0a2/Doc/c-api/type.rst: Path contains '..'
../Python-3.6.0a2/Doc/c-api/typeobj.rst: Path contains '..'
../Python-3.6.0a2/Doc/c-api/unicode.rst: Path contains '..'
../Python-3.6.0a2/Doc/c-api/utilities.rst: Path contains '..'
../Python-3.6.0a2/Doc/c-api/veryhigh.rst: Path contains '..'
../Python-3.6.0a2/Doc/c-api/weakref.rst: Path contains '..'
tar: Error exit delayed from previous errors.
```
[python-build.20160628093022.79796.log.txt](https://github.com/yyuu/pyenv/files/337539/python-build.20160628093022.79796.log.txt)



<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/35568627-3-6-0a2-build-fails-on-os-x-10-11-5-using-python-build-20160130?utm_campaign=plugin&utm_content=tracker%2F282009&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F282009&utm_medium=issues&utm_source=github).
</bountysource-plugin>

--- a/plugins/python-build/share/python-build/3.6.0a2
+++ b/plugins/python-build/share/python-build/3.6.0a2
@@ -1,4 +1,4 @@
 #require_gcc
 install_package "openssl-1.0.2g" "https://www.openssl.org/source/openssl-1.0.2g.tar.gz#b784b1b3907ce39abf4098702dade6365522a253ad1552e267a9a0e89594aa33" mac_openssl --if has_broken_mac_openssl
 install_package "readline-6.3" "http://ftpmirror.gnu.org/readline/readline-6.3.tar.gz#56ba6071b9462f980c5a72ab0023893b65ba6debb4eeb475d7a563dc65cafd43" standard --if has_broken_mac_readline
-install_package "Python-3.6.0a2" "https://www.python.org/ftp/python/3.6.0/Python-3.6.0a2.tgz#66600469e7938b6476e5caaa5a77eafe298877ad81896243970b724c1e6a0401" ldflags_dirs standard verify_py36 ensurepip
+install_package "Python-3.6.0a2" "https://www.python.org/ftp/python/3.6.0/Python-3.6.0a2.tar.xz#2dd2ddf22a63ef9642a20b1f5a40983209e8d7a8ad8371a881fde520edc9ff86" ldflags_dirs standard verify_py36 ensurepip

